api/client/src/main/java/javax/websocket/ContainerProvider.java

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright (c) 2012, 2017 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2012, 2017 Oracle and/or its affiliates and others.
```

Comment:
Not saying this should be changed, but it does bring up a good question on what is the appropriate thing to do here.

Do we update the year?
If we add new classes / files, what/who is the copyright assigned for that new class / file?
etc...

Reply:
The above change (inserting ' and others') is what we are meant to do as we change files as per Eclipse Foundation guidance at  https://www.eclipse.org/projects/handbook/#ip-copyright-headers
This is how the other changes made post donation have been handled.

Reply:
The advice that we were given is that the first year (indicating the year that the original content was created) is enough, and it doesn't need to be updated. It's okay to include the second year, specifying the end of a range of dates. I'd personally be inclined to update the second year if it's included, but defer to the project team if they have different opinions.

The guidance in the handbook regarding adding "and others" indicates what is typically done. It is not hard requirement. 

The PMC may be able to provide more specific guidance.